FWCore/Integration/test/BuildFile.xml

```diff
@@ -440,6 +440,20 @@
     <use name="FWCore/MessageLogger"/>
     <use name="clhep"/>
   </library>
+  <library file="TransformIntProducer.cc, TransformIntStreamProducer.cc" name = "FWCoreIntegrationTransformIntProducer">
+    <flags EDM_PLUGIN="1"/>
+    <use name="FWCore/Framework"/>
+    <use name="FWCore/ParameterSet"/>
+  </library>
+  <test name="testFWCoreIntegrationTransform" command="cmsRun ${LOCALTOP}/src/FWCore/Integration/test/transformTest_cfg.py"/>
+  <test name="testFWCoreIntegrationTransform_onPath" command="cmsRun ${LOCALTOP}/src/FWCore/Integration/test/transformTest_cfg.py -- --onPath"/>
+  <test name="testFWCoreIntegrationTransform_noTransform" command="cmsRun ${LOCALTOP}/src/FWCore/Integration/test/transformTest_cfg.py -- --noTransform"/>
+  <test name="testFWCoreIntegrationTransform_noTransform_onPath" command="cmsRun ${LOCALTOP}/src/FWCore/Integration/test/transformTest_cfg.py -- --noTransform --onPath"/>
+  <test name="testFWCoreIntegrationTransform_stream" command="cmsRun ${LOCALTOP}/src/FWCore/Integration/test/transformTest_cfg.py -- --stream"/>
+  <test name="testFWCoreIntegrationTransform_stream_onPath" command="cmsRun ${LOCALTOP}/src/FWCore/Integration/test/transformTest_cfg.py -- --stream --onPath"/>
+  <test name="testFWCoreIntegrationTransform_noPut" command="cmsRun ${LOCALTOP}/src/FWCore/Integration/test/transformTest_cfg.py -- --noPut || exit 0"/>
```

Comment:
@Dr15Jones , please use `|| true` instead of `|| exit 0` ( `exit` stops scram to run the post unit test part). By the way, what will happen if `cmsRun ${LOCALTOP}/src/FWCore/Integration/test/transformTest_cfg.py -- --noPut` runs successfully? I would suggest to create a shell script e.g
```
> cat transformTest.sh
#!/bin/bash -ex
ecode=$1 ; shift
res=0
cmsRun ${LOCALTOP}/src/FWCore/Integration/test/transformTest_cfg.py -- $@ || res=1
if [ "$res" != "$ecode" ] ; then
  exit 1
fi
```
and then you can use it in buildfile
```
<test name="testFWCoreIntegrationTransform_onPath"      command="fwTransformTest.sh 0 --onPath"/>
<test name="testFWCoreIntegrationTransform_noTransform" command="fwTransformTest.sh 0 --noTransform"/>
...
...
<test name="testFWCoreIntegrationTransform_noPut"       command="fwTtransformTest.sh 1 --noPut"/>
```

Reply:
@smuzaffar would the use of `!` work. I.e. 
```
command="! cmsRun ${LOCALTOP}/src/FWCore/Integration/test/transformTest_cfg.py -- --noPut"
```

that is an allowed operation in bash. It is also exactly logic I need.

Reply:
@Dr15Jones , oh wow, yes that will work . Just need to make sure that there is `space` between `!` and the command